MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly

```diff
@@ -0,0 +1,105 @@
+\version "2.18.2"
+
+\header {
+  title = "Nativity of the Theotokos"
+  subtitle = "Antiphon"
+  subsubtitle = "Verse for 3rd (Sung immediately before Apolytikion)"
+  copyright = "This arrangement is free to distribute for Liturgical use"
+  tagline = "Arranged using https://oca.org/liturgics/learning-the-tones"
+}
+
+#(set-default-paper-size "letter")
+
+% Provide an easy way to group a bunch of text together on a breve
+% http://lilypond.org/doc/v2.18/Documentation/notation/working-with-ancient-music_002d_002dscenarios-and-solutions
+recite = \once \override LyricText.self-alignment-X = #-1
+
+\defineBarLine "invisible" #'("" "" "")
+global = {
+  \time 1/1 % Not used, Time_signature_engraver is removed from layout
+  \key g \major
+  \set Timing.defaultBarType = "invisible" %% Only put bar lines where I say
+}
+
+verseOne = \lyricmode {
+  Here will I dwell, for I have cho -- sen her.
+  The Most High hath hall -- ow -- ed His tab -- er -- na -- cle.
+  Ho -- ly is Thy tem -- ple,
+  won -- der -- ful in righ -- teous -- ness.
+}
+
+soprano = \relative g' {
+  \global
+  % Verse 1
+  g4 g a b b a b c2 c4 b4 r4 \bar "|"
+  g4 a b b b b b g a2 a4 a g2 \bar "||"
+  g4 a b g c2. b2 r4 \bar "|"
+  g4 a b g a2 a2 g2 \bar "|."
+}
+
+alto = \relative g' {
+  \global
+  % Verse 1
+  g4 g g g g g g g2 g4 g4 r4
+  g4 g g g  g g g g fis2 fis d2
```

Comment:
fis2 fis should be fis2 fis4 fis

Reply:
Also remove a g for "-lowed"

Reply:
What do you mean remove a g ?

Reply:
![image](https://user-images.githubusercontent.com/11756408/45196198-f9ccc400-b20f-11e8-8a6f-6659a60e8eb8.png)
?

Reply:
To match the change in the lyrics with hal-lowed as 2 syllables